pkg/infoschema/infoschema_v2.go

```diff
@@ -455,6 +455,28 @@ func (is *infoschemaV2) EvictTable(schema, tbl string) {
 	is.tableCache.Remove(tableCacheKey{itm.tableID, itm.schemaVersion})
 }
 
+type helper struct {
+	end           tableItem
+	schemaVersion int64
+	ok            bool
```

Comment:
ok as a field name is confusing, what does it mean?

Reply:
`ok` here is used to check whether the object is found or not.
Since the return object is a `tableItem` type, not a `*tableItem`, we need this extra flag.

```
var ok bool
var res tableItem

btree.Descend(end, func(item tableItem) {
if .. {
	ok = true
	res = item
})
```

You can view it as 

```
	res, ok := searchTheBtree(...)
	if !ok {
		...
	}
```

Reply:
prefer we name it as `found`, more clear

Reply:
+1 for `found`.

Reply:
Done